src/likelihoods/negativebinomial.jl

```diff
@@ -147,9 +147,10 @@ struct NBParamII{T} <: NBParamMean
 end
 
 function (l::NegativeBinomialLikelihood{<:NBParamII})(f::Real)
-    μ = l.invlink(f)
-    ev = l.params.α * μ
-    return NegativeBinomial(_nb_mean_excessvar_to_r_p(μ, ev)...)
+    # Simplify parameter conversions and avoid splatting
+    r = inv(l.params.α)
+    p = inv(one(l.params.α) + l.params.α * l.invlink(f))
```

Comment:
One could unpack alpha once to reduce the number of field accesses (even though the compiler can optimize it often, I guess).

Reply:
```suggestion
    α = l.params.α
    r = inv(α)
    p = inv(one(α) + α * l.invlink(f))
```